lang/main-ru.json

```diff
@@ -1009,6 +1214,87 @@
         "sendFeedback": "Обратная связь",
         "startMeeting": "Создать конференцию",
         "terms": "Условия",
-        "title": "Защищенная, полнофункциональная и совершенно бесплатная система видеоконференций"
+        "title": "Сервис запущен при содействии компании <a class=\"link-alt\" target=\"_blank\" href=\"https://www.mobiles.by\">ООО “Мобильный сервис”</a>. Работает на базе программного обеспечения с открытым исходным кодом на оборудовании, расположенном в ЦОД РУП “Белтелеком”",
+        "support": "Поддержка",
+        "quickStart": "Рекомендации по использованию",
+        "userGuide": "Руководство пользователя",
+        "addMeetingName": "Добавить название встречи",
+        "jitsiOnMobile": "Navek для мобильных устройств — загрузите наши приложения и начните встречу из любого места",
+        "logo": {
+            "calendar": "Calendar логотип",
+            "desktopPreviewThumbnail": "Миниатюра предварительного просмотра на рабочем столе",
+            "googleLogo": "Google логотип",
+            "logoDeepLinking": "Логотип Navek",
```

Comment:
Navek ? what's 'Navek' ? Jitsi is not a word that can be translated, but 'Navek' does not look like a phonetical transcription - to begin with, these are not Cyrillic characters. With all other languages, even non Latin, 'Jitsi' has been kept as is, except for Arabic where the translator has gone to transcription. But in Arabic, the characters used are non Latin, while it's not the case here. This is plain strange, because in the rest of the file, 'Jitsi meet' has been kept as is (in Latin characters).